src/packagedcode/dockerfileLABEL.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+
+import io
+from pathlib import Path
+from dockerfile_parse import DockerfileParser  
+from packagedcode import models
+from packagedcode import utils
+
+
+class DockerfileHandler(models.DatafileHandler):
+    datasource_id = 'dockerfile'
+    default_package_type = 'docker-image'
+    path_patterns = ('Dockerfile', 'containerfile', '*.dockerfile')  

```

Comment:
We need to test the `is_datafile()` here with examples from here:
* https://github.com/search?q=org.opencontainers.image+path%3A**%2FDockerfile&type=code&ref=advsearch
* https://github.com/search?q=org.opencontainers.image+path%3A**%2F*containerfile*&type=code&ref=advsearch
* https://github.com/search?q=org.opencontainers.image+path%3A**%2F*.dockerfile&type=code&ref=advsearch

Also have these as tests for parsers, choosing from more popular repos/more diverse examples of usage.

See for example other tests like this: https://github.com/aboutcode-org/scancode-toolkit/blob/develop/tests/packagedcode/test_npm.py#L445